Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Install dependencies only when needed
+FROM node:16 AS deps
+WORKDIR /app
+COPY package.json yarn.lock ./
+RUN yarn install --frozen-lockfile
+
+# Rebuild the source code only when needed
+FROM node:16 AS builder
+
+WORKDIR /app
+
+COPY . .
+COPY --from=deps /app/node_modules ./node_modules
+RUN yarn build
+
+# Production image, copy all the files and run next
+FROM node:16 AS runner
+
+ENV NODE_ENV production
+
+WORKDIR /app
+
+RUN addgroup --system --gid 1001 nodejs
+RUN adduser --system --uid 1001 nextjs
+
+COPY --from=builder /app/next.config.js ./
+COPY --from=builder /app/public ./public
+COPY --from=builder --chown=nextjs:nodejs /app/.next ./.next
+COPY --from=builder /app/node_modules ./node_modules
+COPY --from=builder /app/package.json ./package.json
+COPY --from=builder /app/scripts ./scripts
+COPY --from=builder --chown=nextjs:nodejs /app/src/data ./src/data
```

Comment:
This part looks different from https://github.com/CodeForAfrica/TwoopsTrackerUI/blob/b898b727481ec2da62e3dbfc4e93e8faf566fd09/Dockerfile#L38-L46